lib/common/test_utils.go

```diff
@@ -111,6 +89,11 @@ func DropTables(db *sql.DB) {
 		log.Fatalf("an error '%s' was not expected when dropping the table", err)
 	}
 
+	_, err = tx.Exec("DROP TABLE IF EXISTS telemetryRecord")
```

Comment:
Should this table be included in the list on line 31?

Reply:
It needs to be dropped after all other tables and indexes